R/apply_Crosstalk.R

```diff
@@ -0,0 +1,512 @@
+## Based on research paper "A novel tool to assess crosstalk in single-grain luminescence detection",
+##  de Boer et al., 2025
```

Comment:
Please the an appropriate reference filed `#'@references <your reference>` otherwise this information will not be available easily to package users.